docs/roman/datamodels/metadata.rst

```diff
@@ -8,74 +8,44 @@ Metadata information associated with a data model is accessed through
 
 Metadata values are automatically type-checked against the schema when
 they are set. Therefore, setting a attribute which expects a number to a
-string will raise an exception::
-
-.. doctest-skip::
-
-    >>> from romancal.datamodels import ImageModel
-    >>> model = ImageModel()
-    >>> model.meta.target.ra = "foo"    # doctest: +SKIP
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-      File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
-        object.__setattr__(self, attr, val)
-      File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
-        val = self.to_basic_type(val)
-      File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
-        raise ValueError(e.message)
-    ValueError: 'foo' is not of type u'number'
+string will raise an exception.
+
+.. code-block:: python
+
+        from romancal.datamodels import ImageModel
+        model = ImageModel()
+        model.meta.target.ra = "foo"
+        Traceback (most recent call last):
+          File "<stdin>", line 1, in <module>
+          File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
+            object.__setattr__(self, attr, val)
+          File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
+            val = self.to_basic_type(val)
+          File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
+            raise ValueError(e.message)
+        ValueError: 'foo' is not of type u'number'
 
 The set of available metadata elements is defined in a YAML Schema
-that ships with `romancal.datamodels`.
-
-There is also a utility method for finding elements in the metadata
-schema.  `search_schema` will search the schema for the given
-substring in metadata names as well as their documentation.  The
-search is case-insensitive::
-
-.. doctest-skip::
-
-    >>> from romancal.datamodels import ImageModel
-    >>> # Create a model of the desired type
-    >>> model = ImageModel()
-    >>> # Call `search_schema` on it to find possibly related elements.
-    >>> model.search_schema('target')
-    meta.target
-    <BLANKLINE>
-    meta.target.catalog_name
-    <BLANKLINE>
-    meta.target.dec
-    <BLANKLINE>
-    meta.target.dec_uncertainty
-    <BLANKLINE>
-    meta.target.proper_motion_dec
-    <BLANKLINE>
-    meta.target.proper_motion_epoch
-    <BLANKLINE>
-    meta.target.proper_motion_ra
-    <BLANKLINE>
-    meta.target.proposer_dec
-    <BLANKLINE>
-    meta.target.proposer_name
-    <BLANKLINE>
-    meta.target.proposer_ra
-    <BLANKLINE>
-    meta.target.ra
-    <BLANKLINE>
-    meta.target.ra_uncertainty
-    <BLANKLINE>
-    meta.target.source_type
-    <BLANKLINE>
-    meta.target.source_type_apt
-    <BLANKLINE>
-    meta.target.type
-    <BLANKLINE>
-    meta.visit.internal_target
-    <BLANKLINE>
+that is installed with `roman_datamodels` from the `RAD` (Roman Attribute
```

Comment:
A link to the rad repo would be convenient here.

Reply:
Done